trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/edit-payment-helper.ts

```diff
@@ -67,6 +69,7 @@ export const getEditPaymentViewModel = (
   paymentId: string,
   isCheckboxChecked: (checkboxId: string) => boolean,
   errors: EditPaymentErrorsViewModel = EMPTY_PAYMENT_ERRORS_VIEW_MODEL,
+  redirectTab: ReconciliationForReportTab | undefined = undefined,
```

Comment:
Do we need to do `undefined = undefined`, could it just be the following?:

```suggestion
  redirectTab?: ReconciliationForReportTab
```

Reply:
Sorry, missed this from Josh's initial work - resolved this now thanks